diff --git src/themes/notnaked/router/index.js src/themes/notnaked/router/index.js
index 5e3b1cd6..d46df11f 100644
--- src/themes/notnaked/router/index.js
+++ src/themes/notnaked/router/index.js
@@ -1,7 +1,7 @@
 const Home = () =>
-  import(/* webpackChunkName: "vsf-home" */ 'theme/pages/Home');
+  import(/* webpackChunkName: "vsf-home" */ 'src/modules/peregrine/pages/Home.vue');
 const Static = () =>
-  import(/* webpackChunkName: "vsf-static" */ 'theme/pages/Static');
+  import(/* webpackChunkName: "vsf-static" */ 'src/modules/peregrine/pages/Static.vue');
 const ErrorPage = () =>
   import(/* webpackChunkName: "vsf-error" */ 'theme/pages/Error');
 const Product = () =>
@@ -16,7 +16,7 @@ const Wishlist = () =>
   import(/* webpackChunkName: "vsf-my-account" */ 'theme/pages/Wishlist');
 
 let routes = [
-  { name: 'home', path: '/', component: Home, alias: '/pwa.html' },
+  { name: 'home', path: '/', component: Static, alias: '/pwa.html' },
   { name: 'checkout', path: '/checkout', component: Checkout, meta: { layout: 'minimal' } },
   { name: 'legal', path: '/legal', component: Static },
   { name: 'privacy', path: '/privacy', component: Static },