diff --git src/themes/notnaked/config/modules.ts src/themes/notnaked/config/modules.ts
index c0483db..18a4697 100644
--- src/themes/notnaked/config/modules.ts
+++ src/themes/notnaked/config/modules.ts
@@ -14,6 +14,7 @@ import { PaymentBackendMethodsModule } from 'src/modules/payment-backend-methods
 import { PaymentCashOnDeliveryModule } from 'src/modules/payment-cash-on-delivery'
 
 import { registerModule } from '@vue-storefront/core/lib/modules'
+import { PeregrineModule } from 'src/modules/peregrine'
 
 // TODO:distributed across proper pages BEFORE 1.11
 export function registerClientModules () {
@@ -30,6 +31,7 @@ export function registerClientModules () {
   registerModule(CmsModule)
   registerModule(NewsletterModule)
   registerModule(WishlistModule)
+  registerModule(PeregrineModule)
 }
 
 // Deprecated API, will be removed in 2.0
diff --git src/themes/notnaked/router/index.js src/themes/notnaked/router/index.js
index 5e3b1cd..244228f 100644
--- src/themes/notnaked/router/index.js
+++ src/themes/notnaked/router/index.js
@@ -1,7 +1,9 @@
 const Home = () =>
-  import(/* webpackChunkName: "vsf-home" */ 'theme/pages/Home');
+  import(/* webpackChunkName: "vsf-home" */ 'src/modules/peregrine/pages/Home.vue');
 const Static = () =>
-  import(/* webpackChunkName: "vsf-static" */ 'theme/pages/Static');
+  import(/* webpackChunkName: "vsf-static" */ 'src/modules/peregrine/pages/Static.vue');
+const Blog = () =>
+  import(/* webpackChunkName: "vsf-static" */ 'src/modules/peregrine/pages/Blog.vue');
 const ErrorPage = () =>
   import(/* webpackChunkName: "vsf-error" */ 'theme/pages/Error');
 const Product = () =>
@@ -16,7 +18,7 @@ const Wishlist = () =>
   import(/* webpackChunkName: "vsf-my-account" */ 'theme/pages/Wishlist');
 
 let routes = [
-  { name: 'home', path: '/', component: Home, alias: '/pwa.html' },
+  { name: 'home', path: '/', component: Static, alias: '/pwa.html' },
   { name: 'checkout', path: '/checkout', component: Checkout, meta: { layout: 'minimal' } },
   { name: 'legal', path: '/legal', component: Static },
   { name: 'privacy', path: '/privacy', component: Static },
@@ -39,7 +41,8 @@ let routes = [
   { name: 'configurable-product', path: '/p/:parentSku/:slug/:childSku', component: Product },
   { name: 'product', path: '/p/:parentSku/:slug/:childSku', component: Product },
   { name: 'category', path: '/c/:slug', component: Category },
-  { name: 'cms-page', path: '/i/:slug', component: Static }
+  { name: 'cms-page', path: '/i/:slug', component: Static },
+  { name: 'collections', path: '/collections/:id', component: Static }
 ];
 
 export default routes;
